<?xml version="1.0" encoding="UTF-8"?>
<!-- Path: gui\informacao_produto\add_historical_version_dialog.ui -->
<ui version="4.0">
 <class>AddHistoricalVersionDialog</class>
 <widget class="QDialog" name="AddHistoricalVersionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Versão Histórica ao Produto</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="versionInfoGroupBox">
     <property name="title">
      <string>Informações da Versão Histórica</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="nomeVersaoLabel">
          <property name="text">
           <string>Nome da Versão:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nomeVersaoLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="versaoLabel">
          <property name="text">
           <string>Número da Versão:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="versaoLineEdit">
          <property name="placeholderText">
           <string>Ex: 1.0, 2ª Edição</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="subtipoProdutoLabel">
          <property name="text">
           <string>Subtipo de Produto:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="subtipoProdutoComboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="loteLabel">
          <property name="text">
           <string>Lote (opcional):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="loteComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="orgaoProdutorLabel">
          <property name="text">
           <string>Órgão Produtor:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="orgaoProdutorLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="palavrasChaveLabel">
          <property name="text">
           <string>Palavras-chave:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="palavrasChaveLineEdit">
          <property name="placeholderText">
           <string>Separadas por vírgula</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="dataCriacaoLabel">
          <property name="text">
           <string>Data de Criação:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dataCriacaoDateEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="dataEdicaoLabel">
          <property name="text">
           <string>Data de Edição:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dataEdicaoDateEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="descricaoLabel">
        <property name="text">
         <string>Descrição:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="descricaoTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="metadadoLabel">
        <property name="text">
         <string>Metadados (JSON):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="metadadoTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>